### PR TITLE
refactor: drop dialog module

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/cdk/schematics/ng-add/tests/schematic-ng-add.spec.ts [hunk 1]
```diff
@@ -254,13 +254,13 @@ describe('ng-add', () => {
...
             `import { NgDompurifySanitizer } from "@taiga-ui/dompurify";
-import { TuiRoot, TuiDialogModule, TuiAlertModule, TUI_SANITIZER } from "@taiga-ui/core";
+import { TuiRoot, TuiDialog, TuiAlert, TUI_SANITIZER } from "@taiga-ui/core";
 import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
 import {NgModule} from '@angular/core';
 import {AppComponent} from './app.component';
 
 @NgModule({declarations: [AppComponent],
-    imports: [BrowserAnimationsModule, TuiRoot, TuiDialogModule, TuiAlertModule],
+    imports: [BrowserAnimationsModule, TuiRoot, TuiDialog, TuiAlert],
```
Comment (waterplea) [inline projects/cdk/schematics/ng-add/tests/schematic-ng-add.spec.ts:263]: We no longer need dialog and alert in main module

**Review by waterplea — APPROVED** [review]

File: projects/cdk/schematics/ng-add/tests/schematic-ng-add.spec.ts (hunk 1 above)
Comment (splincode) [reply projects/cdk/schematics/ng-add/tests/schematic-ng-add.spec.ts]: fixed

**Review by nsbarsukov — APPROVED** [review]

File: projects/cdk/schematics/ng-add/schema.ts [hunk 2]
```diff
@@ -1,6 +1,4 @@
 export interface TuiSchema {
-    readonly addAlertModule: boolean;
-    readonly addDialogsModule: boolean;
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-add/schema.ts:3]: Fix unit tests

```
projects/cdk/schematics/ng-add/tests/schematic-ng-add.spec.ts:42:13 - error TS2322: Type '{ addSanitizer: false; addGlobalStyles: false; addDialogsModule: boolean; addAlertModule: boolean; addons: never[]; project: string; 'skip-logs': boolean; }' is not assignable to type 'TuiSchema'.
  Object literal may only specify known properties, and 'addDialogsModule' does not exist in type 'TuiSchema'.

42             addDialogsModule: false,
               ~~~~~~~~~~~~~~~~~~~~~~~

projects/cdk/schematics/ng-add/tests/schematic-ng-add.spec.ts:68:13 - error TS2322: Type '{ addSanitizer: true; addGlobalStyles: false; addDialogsModule: boolean; addAlertModule: boolean; addons: string[]; project: string; 'skip-logs': boolean; }' is not assignable to type 'TuiSchema'.
  Object literal may only specify known properties, and 'addDialogsModule' does not exist in type 'TuiSchema'.

68             addDialogsModule: false,
               ~~~~~~~~~~~~~~~~~~~~~~~

projects/cdk/schematics/ng-add/tests/schematic-ng-add.spec.ts:98:13 - error TS2322: Type '{ addSanitizer: true; addGlobalStyles: true; addDialogsModule: boolean; addAlertModule: boolean; addons: string[]; project: string; 'skip-logs': boolean; }' is not assignable to type 'TuiSchema'.
  Object literal may only specify known properties, and 'addDialogsModule' does not exist in type 'TuiSchema'.

98             addDialogsModule: false,
               ~~~~~~~~~~~~~~~~~~~~~~~

projects/cdk/schematics/ng-add/tests/schematic-ng-add.spec.ts:2[46](https://github.com/taiga-family/taiga-ui/actions/runs/9597563783/job/26466893084?pr=7870#step:5:47):13 - error TS2322: Type '{ addSanitizer: true; addGlobalStyles: false; addDialogsModule: boolean; addAlertModule: boolean; addons: never[]; project: string; 'skip-logs': boolean; }' is not assignable to type 'TuiSchema'.
  Object literal may only specify known properties, and 'addDialogsModule' does not exist in type 'TuiSchema'.

246             addDialogsModule: true,
```

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/cdk/schematics/ng-add/schema.ts (hunk 2 above)
Comment (splincode) [reply projects/cdk/schematics/ng-add/schema.ts:3]: fixed

**Pull request merged**